core/app/modules/kit-library/assets/js/components/preview-responsive-controls.js

```diff
@@ -0,0 +1,30 @@
+import { Button, Grid } from '@elementor/app-ui';
+import { breakpoints } from '../pages/preview';
+
+import './preview-responsive-controls.scss';
+
+export default function PreviewResponsiveControls( props ) {
+	return (
+		<Grid container alignItems="center" justify="center" className="e-kit-library__preview-responsive-controls">
+			{ breakpoints.map( ( { label, value } ) => (
```

Comment:
@shilo-ey  does it should work with the custom breakpoints?

Reply:
No, only the 3 default ones